tools/building.py

```diff
@@ -304,11 +304,8 @@ def has_substr(args, substr):
   return (args, env)
 
 
-def configure(args, stdout=None, stderr=None, env=None, cflags=[], **kwargs):
-  if env:
-    env = env.copy()
-  else:
-    env = get_building_env(cflags=cflags)
+def configure(args):
```

Comment:
Is this a relevant change? (I can't see any callsites for this function in this PR) If other functions are calling configure using `emconfigure`, where is this function used now?

Reply:
The function has only one remaining callsite in `emconfigure.py` which only passed one argument..  hence we can remove all the others.  yay!

Same for `make` below.

The followup change actually inlines these two functions.